Bibliotheque nationale de France.js

```diff
@@ -565,7 +596,10 @@ var BnfClass = function() {
 					}
 				}
 			}
-				
+			// case last zone
+			if(tag) {
+						record.addField(tag, ind, content);
+					}	
```

Comment:
I have made this. because in the loop, the last line was not processed.
I take a code and I do not chang it too much